backend/tests/documents/test_collaborator_access.py

```
@@ -0,0 +1,137 @@
+from datarooms.models import Dataroom, DataroomDocument, DataroomCollaborator
+from documents.models import Document, DocumentVersion
+
+User = get_user_model()
+
+
+@pytest.fixture
+def other_user(db, organization):
+    return User.objects.create_user(
+        username="collab@example.com",
+        email="collab@example.com",
+        password="password123",
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "password"

(S106)